src/components/ground-nav/ground-nav.twig

```diff
@@ -100,9 +180,26 @@
         </ul>
       </nav>
 
-      {% if kudos %}
-        {{ kudos }}
-      {% endif %}
+      <div class="c-ground-nav__social">
+        <h2 class="u-hidden-visually">
+          Social
+        </h2>
+        <ul class="c-ground-nav__social-list o-list o-list--inline" role="list">{# 5 #}
```

Comment:
We can follow the same [pattern described above](https://github.com/cloudfour/cloudfour.com-patterns/pull/2194/files#r1309158218) here for the same reasons. 

Maybe even change the heading from "Social" to "Social links"? That way, the screen reader reads, "list Social links, 5 items". 🙂 

<img width="657" alt="Screen Shot 2023-08-29 at 10 46 55 AM" src="https://github.com/cloudfour/cloudfour.com-patterns/assets/459757/a2213d81-580b-473e-927c-9a5121e5f26d">


```suggestion
        <h2 id="social-links-heading" hidden>
          Social links
        </h2>
        <ul aria-labelledby="social-links-heading" class="c-ground-nav__social-list o-list o-list--inline" role="list">{# 5 #}
```

Reply:
Done!